fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResource.java

```diff
@@ -649,8 +639,11 @@ public String retrieveAll(@Context final UriInfo uriInfo,
     @POST
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String calculateLoanScheduleOrSubmitLoanApplication(@QueryParam("command") final String commandParam,
-            @Context final UriInfo uriInfo, final String apiRequestBodyAsJson) {
+    @ApiOperation(value = "Calculate loan repayment schedule", httpMethod = "POST", notes = "It calculates the loan repayment Schedule")
```

Comment:
add mandatory all mandatory fields here

Reply:
Can you tell me how to add mandatory?

Reply:
add in value = "Submit a new Loan Application" and other details for the same 
we have to different docs according to the query param
so do mention all

Reply:
Not correct

Reply:
add in value = "Submit a new Loan Application | Calculate loan repayment schedule" and other details for the same
we have two different docs according to the query param
so do mention all
and in the description: "Mandatory Fields for Submit a new Loan Application : clientId, productId, principal, loanTermFrequency, loanTermFrequencyType, loanType, numberOfRepayments, repaymentEvery, repaymentFrequencyType, interestRatePerPeriod, amortizationType, interestType, interestCalculationPeriodType, transactionProcessingStrategyId, expectedDisbursementDate, submittedOnDate, loanType Additional Mandatory Fields if interest recalculation is enabled for product and Rest frequency not same as repayment period
recalculationRestFrequencyDate
Additional Mandatory Fields if interest recalculation with interest/fee compounding is enabled for product and compounding frequency not same as repayment period
recalculationCompoundingFrequencyDate
Additional Mandatory Field if Entity-Datatable Check is enabled for the entity of type loan.
datatables" + "\n\n\n\n\n" + 


"Mandatory Fields for Calculate loan repayment schedule are: productId, principal, loanTermFrequency, loanTermFrequencyType, numberOfRepayments, repaymentEvery, repaymentFrequencyType, interestRatePerPeriod, amortizationType, interestType, interestCalculationPeriodType, expectedDisbursementDate, transactionProcessingStrategyId"

Reply:
edit accordingly with new space characters so that it will not create a mess in Swagger-UI
so add new line character"\n" accordingly and refer to the current documentation for more details